### DISCUSSION TITLE
Не запускается

### DISCUSSION THREAD

**Discussion by DjackVorobeu** [LostRuins/koboldcpp#113] [General]
Ничего не могу сделать, выбивает эту ошубку
![0UCPn7UMQwc](https://user-images.githubusercontent.com/117567002/233856816-658e1017-becc-4592-938f-6ddb1af9406a.jpg)

**Comment by LostRuins** [comment]
Do you have Python installed? If you already have python installed, you can try running the Python script directly instead of using PyInstaller. You can download the Zip archive and extract it from this link: https://github.com/LostRuins/koboldcpp/releases/download/v1.12/koboldcpp.zip 

---

У вас установлен питон? Если у вас уже установлен Python, вы можете попробовать запустить скрипт Python напрямую, а не с помощью PyInstaller. Вы можете скачать Zip-архив и распаковать его по этой ссылке: https://github.com/LostRuins/koboldcpp/releases/download/v1.12/koboldcpp.zip
